konsave: add page (#9773)
tldr-pages/tldr: pages/linux/konsave.md

# konsave

> > A CLI program that lets you save and apply your Linux customizations with just one command!

Reviewer: ```suggestion
> Save and apply your Linux customizations.
```

> More information: <https://github.com/Prayag2/konsave>.

- Save the current configuration as a profile:

`konsave --save {{profile_name}}`

- Apply a profile:

`konsave --apply {{profile_name}}`

- Save the current configuration as a profile, overwriting existing profiles if they exist with the same name:

`konsave -s {{profile_name}} --force`

- List all profiles:

`konsave --list`

- Remove a profile:

`konsave --remove {{profile_name}}`

- Export a profile as a `.knsv` to the home directory:

`konsave --export-profile {{profile_name}}`

- Import a `.knsv` profile:

> `konsave --import-profile {{path to the file}}`

Reviewer: I'm not sure if it should be `profile_name.knsv`. Feel free to replace it base on your knowledge.
```suggestion
`konsave --import-profile {{path/to/profile_name.knsv}}`
```

Reviewer 2: Why not `profile.knsv`?

Reviewer: Because all other examples use `profile_name` as the placeholder.

Author: So Should I Change it ?

Reviewer 2: > So Should I Change it ?

Yeah

# konsave

> > A CLI program that will let you save and apply your Linux customizations with just one command!

Reviewer 2: ```suggestion
> A CLI program that lets you save and apply your Linux customizations with just one command!
```

Reviewer 3: The exclamation point is not required.

> More information: <https://github.com/Prayag2/konsave>.

> - Save current configuration as a profile:

Reviewer 2: ```suggestion
- Save the current configuration as a profile:
```

`konsave --save <profile name>`

- Apply a profile:

`konsave --apply <profile name>`

- Overwrite an already saved profile:

`konsave -s <profile name> --force`

- List all profiles:

`konsave --list`

- Remove a profile:

`konsave --remove <profile name>`

> - Export a profile as a ".knsv":

Reviewer 2: ![image](https://user-images.githubusercontent.com/26346867/212459949-a95ec257-1a0a-4ded-a8e6-5ada2eae3599.png)

Checked the code, seems like it is exported to the home directory.

> - Export a profile as a ".knsv":

Reviewer 3: Then my suggestion has been edited to include this.

`konsave --export-profile <profile name>`

- Import a ".knsv" file:

> `konsave --import-profile <path to the file>`

Reviewer 3: ```suggestion
`konsave --import-profile {{path/to/profile.knsv}}`
```
